spl/Cargo.toml

```diff
@@ -26,9 +25,15 @@ token_2022 = ["spl-token-2022"]
 anchor-lang = { path = "../lang", version = "0.29.0", features = ["derive"] }
 borsh = { version = ">=0.9, <0.11", optional = true }
 mpl-token-metadata = { version = "3.1.0", optional = true }
-serum_dex = { git = "https://github.com/openbook-dex/program/", rev = "1be91f2", version = "0.4.0", features = ["no-entrypoint"], optional = true }
+serum_dex = { git = "https://github.com/openbook-dex/program/", rev = "1be91f2", version = "0.4.0", features = [
+    "no-entrypoint",
+], optional = true }
 solana-program = "1.16"
-spl-associated-token-account = { version = "2.2", features = ["no-entrypoint"], optional = true }
+spl-associated-token-account = { version = "2.2", features = [
+    "no-entrypoint",
+], optional = true }
 spl-memo = { version = "4", features = ["no-entrypoint"], optional = true }
 spl-token = { version = "4", features = ["no-entrypoint"], optional = true }
-spl-token-2022 = { version = "0.9", features = ["no-entrypoint"], optional = true }
+spl-token-2022 = { version = "0.9", features = [
+    "no-entrypoint",
+], optional = true }
```

Comment:
I can see this is a long line but this is a very weird way to format. Main point of TOML is single line entries so we could convert this to `[dependencies.<NAME>]` to keep it short but I think it's better to be consistent and revert this change to keep it one line.

Reply:
Yeah sorry automatic formatting